src/status_im/utils/build.clj

```diff
@@ -0,0 +1,10 @@
+(ns status-im.utils.build
+  (:require [cljs.analyzer :as ana]))
```

Comment:
let's use `analyzer` instead of `ana` here

Reply:
Done